deploy/bundle/windows-machine-config-operator/manifests/windows-machine-config-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+            - persistentvolumeclaims
+            - events
+            - configmaps
+            - secrets
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+          - apiGroups:
+            - apps
+            resources:
+            - deployments
+            - daemonsets
+            - replicasets
+            - statefulsets
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+          - apiGroups:
+            - monitoring.coreos.com
+            resources:
+            - servicemonitors
+            verbs:
+            - get
+            - create
+          - apiGroups:
+            - apps
+            resourceNames:
+            - windows-machine-config-operator
+            resources:
+            - deployments/finalizers
+            verbs:
+            - update
+          - apiGroups:
+            - ""
+            resources:
+            - pods
+            verbs:
+            - get
+          - apiGroups:
+            - apps
+            resources:
+            - replicasets
+            - deployments
+            verbs:
+            - get
+          - apiGroups:
+            - wmc.openshift.io
+            resources:
+            - '*'
+            verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+      deployments:
+        - name: windows-machine-config-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: windows-machine-config-operator
+            template:
+              metadata:
+                labels:
+                  name: windows-machine-config-operator
+              spec:
+                containers:
+                  - image: $OPERATOR_IMAGE
+                    imagePullPolicy: Always
+                    name: windows-machine-config-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: 'metadata.annotations[''olm.targetNamespaces'']'
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: windows-machine-config-operator
+                nodeSelector:
+                  beta.kubernetes.io/os: linux
+                serviceAccountName: windows-machine-config-operator
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: false
```

Comment:
We don't need to support all these install modes. We just need the mode that allows us to watch our own namespace.